### PR TITLE
Timestrip plan Inspection

### LINKED ISSUE #3832: Support Inspection of Activities from Plans and Time Views
Clicking on an activity in a plan or time view should reveal additional information about that activity in the inspector. 

This should make use of [the Selection API](https://github.com/nasa/openmct/blob/master/src/selection/Selection.js).

Some design is necessary to determine exactly what information is shown in the inspector on selection. We should also consider what occurs in the event of multiple activities being selected.

### PR DESCRIPTION
**Pull request by charlesh88**
Adds Activity details for plans as per the design below.
Also adds a duration utility to show durations > 1 day. Refactors duration used in Conductor.vue into this utility file.

Resolves #3832

Here are some screenshots of what the Inspector should look like for the two different display cases:

One Activity selected
Screen Shot 2021-05-06 at 3 39 57 PM

Two or more Activities selected
Screen Shot 2021-05-06 at 3 37 29 PM

### CODE REVIEW

File: src/ui/inspector/Properties.vue [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <template>
```
Comment (jvigliotta) [inline src/ui/inspector/Properties.vue:1]: Open MCT copyright stuff if necessary.

File: src/utils/duration.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+const ONE_MINUTE = 60 * 1000;
```
Comment (jvigliotta) [inline src/utils/duration.js:1]: Open MCT Copyright stuff if necessary.

and... Cool! :)

File: src/utils/duration.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+const ONE_DAY = ONE_HOUR * 24;
+
+function normalizeAge(num) {
+    const hundredtized = num * 100;
+    const isWhole = hundredtized % 100 === 0;
+
+    return isWhole ? hundredtized / 100 : num;
+}
+
+function toDoubleDigits(num) {
```
Comment (jvigliotta) [inline src/utils/duration.js:12]: Nice!

File: src/utils/duration.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+        if (age !== 1) {
+            result += 's';
+        }
+    }
+
+    return result;
+}
+
+export function getPreciseDuration(numericDuration) {
```
Comment (jvigliotta) [inline src/utils/duration.js:50]: Nice!

**Review by jvigliotta — CHANGES_REQUESTED** [review]
Looks good! Just a couple items, I'm not positive on the reqs for including copyright stuff... but made notes.

File: src/plugins/plan/Plan.vue [hunk 5]
```diff
@@ -456,6 +466,9 @@ export default {
 
                 const textNode = document.createTextNode(line);
                 textElement.appendChild(textNode);
+                textElement.addEventListener('click', (event) => {
```
[the remark this answers is not in the document]
Comment (shefalijoshi) [reply src/plugins/plan/Plan.vue:469]: Same response :)

File: src/plugins/plan/inspector/PlanActivitiesView.vue [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+
+ Open MCT includes source code licensed under additional open source
+ licenses. See the Open Source Licenses file (LICENSES.md) included with
+ this source code distribution or the Licensing information page available
+ at runtime from the About dialog for additional information.
+-->
+<template>
+<div class="c-inspector__properties c-inspect-properties">
+    <plan-activity-view v-for="(activity, index) in activities"
+                        :key="index"
```
[the remark this answers is not in the document]
Comment (shefalijoshi) [reply src/plugins/plan/inspector/PlanActivitiesView.vue:25]: Ah true. I'll fix that.

[the remark this answers is not in the document]
Comment (shefalijoshi) [reply src/plugins/plan/inspector/PlanActivitiesView.vue]: Done.

File: src/plugins/plan/inspector/PlanActivityView.vue [hunk 7]
```diff
@@ -0,0 +1,58 @@
+<template>
+<div v-if="timeProperties.length"
+     class="u-contents"
+>
+    <div class="c-inspect-properties__header">
+        {{ heading }}
+    </div>
+    <ul v-for="(timeProperty, index) in timeProperties"
+        :key="index"
```
[the remark this answers is not in the document]
Comment (shefalijoshi) [reply src/plugins/plan/inspector/PlanActivityView.vue]: Done.

File: src/utils/duration.js (hunk 2 above)
Comment (shefalijoshi) [reply src/utils/duration.js:23]: Done.

File: src/plugins/plan/inspector/ActivityProperty.vue [hunk 8]
```diff
@@ -0,0 +1,30 @@
+<template>
```
[the remark this answers is not in the document]
Comment (shefalijoshi) [reply src/plugins/plan/inspector/ActivityProperty.vue:23]: Done.

File: src/plugins/plan/inspector/PlanActivitiesView.vue [hunk 9]
```diff
@@ -0,0 +1,288 @@
...
+                        :activity="activity"
+                        :heading="heading"
+    />
+    <!-- If two or more Activities are selected -->
+
+</div>
+    <!--        <div class="c-inspect-properties__header">-->
```
[the remark this answers is not in the document]
Comment (shefalijoshi) [reply src/plugins/plan/inspector/PlanActivitiesView.vue]: Done.

**Review by jvigliotta — APPROVED** [review]
Nice work!

### Reviewer Checklist
Check | Passed
-- | --
Changes appear to address issue?? | Y
Appropriate unit tests included? | N/A
Code style and in-line documentation are appropriate? | Y
Commit messages meet standards? | Y
Has associated issue been labelled unverified? (if PR closes issue) | Y

**Pull request merged**